ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Overview/CVEsTableContainer.tsx

```diff
@@ -73,6 +73,7 @@ function CVEsTableContainer({ defaultFilters, countsData }: CVEsTableContainerPr
                         unfilteredImageCount={imageCountData?.imageCount || 0}
                         getSortParams={getSortParams}
                         isFiltered={isFiltered}
+                        filteredSeverities={searchFilter.Severity as VulnerabilitySeverityLabel[]}
```

Comment:
Nit: we should see if these are easy to replace with the parsed searchFilter as a follow up so that we can avoid the type assertions.